src/ServicePulse.Host/vue/src/components/DataView.vue

```diff
@@ -0,0 +1,173 @@
+<script setup>
+import { computed, onMounted, onUnmounted, ref } from "vue";
+import { useFetchFromServiceControl } from "../composables/serviceServiceControlUrls";
+
+const props = defineProps({
+  apiUrl: String,
+  itemsPerPageOptions: {
+    type: Array,
+    default: () => [20, 35, 50, 75],
+  },
+  itemsPerPage: {
+    type: Number,
+    default: 50,
+  },
+  autoRefresh: {
+    type: Number,
+    default: 0,
```

Comment:
i feel that the default should be null, since 0 is confusing and could be interpreted as being a constant refresh

Reply:
Done